CHANGELOG

```diff
@@ -0,0 +1,51 @@
+# Changelog
+
+Items starting with `DEPRECATE` are important deprecation notices. For more
+information on the list of deprecated flags and APIs please have a look at
+https://docs.docker.com/engine/deprecated/ where target removal dates can also
+be found.
+
+## 17.10.0-ce (2017-10-DD)
+
+### Builder
+
+* Reset uid/gid to 0 in build context to fix cache busting issues on `ADD/COPY` [docker/cli#513](https://github.com/docker/cli/pull/513)
+- Fix build with `ADD` urls without any sub path [moby/moby#34217](https://github.com/moby/moby/pull/34217)
+- Fix loader error with different build syntax [docker/cli#544](https://github.com/docker/cli/pull/544)
+
+### Client
+
+* Move output of `docker stack rm` to stdout [docker/cli#491](https://github.com/docker/cli/pull/491)
+* Remove deprecated `--enable-api-cors` flag [moby/moby#34821](https://github.com/moby/moby/pull/34821)
+* Use nature sort secrets and configs in cli [docker/cli#307](https://github.com/docker/cli/pull/307)
+* Use non-detached mode as default for `docker service` commands [docker/cli#525](https://github.com/docker/cli/pull/525)
+* Set APIVersion on the client, even when Ping fails [docker/cli#546](https://github.com/docker/cli/pull/546)
+
+### Logging
+
+* Move pkg/jsonlog to be a subpackage of the single consumer [moby/moby#34946](https://github.com/moby/moby/pull/34946)
+- Fixed `raw` mode splunk logger [moby/moby#34520](https://github.com/moby/moby/pull/34520)
```

Comment:
This was back ported to 17.09 through https://github.com/docker/docker-ce/pull/233, so not new in 17.10